packages/deployments/contracts/contracts/core/connext/helpers/RelayerProxy.sol

```diff
@@ -366,4 +442,22 @@ contract RelayerProxy is ProposedOwnable, ReentrancyGuard, GelatoRelayFeeCollect
     emit Keep3rChanged(_keep3r, address(keep3r));
     keep3r = IKeep3rV2(_keep3r);
   }
+
+  function _validateProposeSignature(
+    bytes32 _aggregateRoot,
+    uint256 _rootTimestamp,
+    bytes memory _signature
+  ) internal view {
+    // Get the payload
+    bytes32 payload = keccak256(abi.encodePacked(_aggregateRoot, _rootTimestamp));
+    // Recover signer
+    address signer = payload.toEthSignedMessageHash().recover(_signature);
+    if (!spokeConnector.allowlistedProposers(signer)) {
+      revert RelayerProxy__validateProposeSignature_notProposer(signer);
+    }
+  }
+
+  function _proposeAggregateRootCooledDown() internal view returns (bool) {
+    return block.timestamp > (lastProposeAggregateRootAt + proposeAggregateRootCooldown);
```

Comment:
```suggestion
    return block.timestamp >= (lastProposeAggregateRootAt + proposeAggregateRootCooldown);
```
It should be greater or equal otherwise it would always be `proposeAggregateRootCooldown` + 1.

Reply:
Done